grpc/src/main/java/com/linecorp/armeria/server/grpc/UnaryServerCall.java

```diff
@@ -180,7 +181,7 @@ void doClose(Status status, Metadata metadata, boolean completed) {
 
     @Nullable
     @Override
-    O firstResponse() {
+    public O firstResponse() {
```

Comment:
```suggestion
    protected O firstResponse() {
```

Reply:
Done in [63e0b79](https://github.com/line/armeria/pull/4724/commits/63e0b79cdf01d14c58d916c29c87d1f129f255a8)!